test/suite/test_txn07.py

```diff
@@ -217,6 +210,14 @@ def test_ops(self):
         csmall = stat_cursor[stat.conn.log_compress_small][2]
         stat_cursor.close()
 
+        #
```

Comment:
Running the wt utility before collecting the statistics, the stats are reset. The connection is getting closed as part of running the wt utility is the cause of the problem. Moving it down after collecting the statistics doesn't have functionality problem and it fixes the test.

@sulabhM Do you have any idea?

Reply:
I do not know why this changed, there hasn't been any changes to printlog. Stats collection should work as before.

Reply:
I understood the reason behind passing the test before durable history. Earlier also the connection is getting closed and opened but as part of the connection open, WT will create the LAS file, due to which the log records are written and those are compressed and stats are updated. With DH, there are no las file create and it leads to no log records and it fails. Moving the `runWT` later after stats collection is proper way to fix the test and its expectation.